django/test/runner.py

```diff
@@ -101,6 +101,43 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
+class TestCaseDTO:
+    """
+    The ParallelTestSuite requires that all test cases be pickleable. When a
+    SubTest fails with an exception, it's pickled and shipped back to the main
+    process to be displayed to the user. Unfortunately, Django's TestCases
+    aren't always pickleable (for example, they may contain an instance of
+    django.test.Client, which can't be cleanly pickled if an exception was
+    raised by the view).
+
+    Luckily the outer TestRunner only cares about a small subset of the fields
+    on TestCase, and those are usually pickleable. We use this object to pluck
+    out those fields, discarding extra unpickleable members.
```

Comment:
This docstring spends a lot of time explainining the problem. I think it can be edited down to something a little more terse to make the class more maintainable. Perhaps:

> Represents a failed test or SubTest for use in parallel testing. Needed as not all attributes of a `TestCase` may be pickleable, so this class extracts only those needed to report the error in the main process.